packages/installer/upgrade.sh

```diff
@@ -124,7 +124,7 @@ upgrade() {
     export HOME=/tmp # Docker Content Trust data is stored in $HOME/.docker. We don't need those to persist
 
     if [ -n "$CHANNEL_UPGRADES" ] && [ "$CHANNEL_UPGRADES" == true ]; then
-        luet install --system-target /tmp/upgrade --system-engine memory -y $UPGRADE_IMAGE
```

Comment:
There is no way to opt-out from verification right? I think we should respect `--verify` here and disable/enable the plugin loading on request

Reply:
yep, good pointer! I was wondering how to add tests to verify this and just stumble upon this, there is currently no switch to enable/disable this and that is bad. Let me update this with the verify switch on place for this!

Reply:
done!